libraries/Microsoft.Bot.Builder.AI.QnA/Models/KnowledgeBaseAnswer.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA.Models
+{
+    /// <summary>
+    /// Represents an individual result from a knowledge base query.
+    /// </summary>
+    public class KnowledgeBaseAnswer
```

Comment:
This class seems to be used internally to connect to the service and it is not needed outside, consider making it internal